docs/platform-azure.md

```diff
@@ -34,7 +34,7 @@ This will output a `azure.vhd` image.
 To deploy the `azure.vhd` image on Azure, invoke the following command:
 
 ```
-linuxkit run azure --resourceGroupName <resource-goup-name> --accountName <storageaccountname> --location westeurope <path-to-your-azure.vhd>
+linuxkit run azure --resource-group <resource-goup-name> --storage-account <storage-account-name> --location westeurope <path-to-your-azure.vhd>
```

Comment:
maybe i don't understand what you mean. I left the `name` bit in the `<...>` but removed it from the command line option. Leaving it in the `<...>` makes it clearer that it should be the name